chp00_introduction/NOC_I_01_RandomWalkTraditional/index.html

```diff
@@ -1,7 +1,8 @@
 <head>
-  <script language="javascript" type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/p5.js/0.5.7/p5.js"></script>
+  <script language="javascript" type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.5.7/p5.js"></script>
```

Comment:
I think technically leaving out `https` is better this way the example can work from either `http` or `https` pages?